<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGry</class>
 <widget class="QMainWindow" name="OknoGry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yahtzee - Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QLabel" name="labelTura">
        <property name="text">
         <string>Teraz tura: -</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:18px;font-weight:800;color:#07a;</string>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="horizontalSpacerTop1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <!-- Panel sieci -->
      <item>
       <widget class="QLineEdit" name="editIp">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>IP serwera</string>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QLineEdit" name="editPort">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Port</string>
        </property>
        <property name="text">
         <string>5000</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="btnHost">
        <property name="text">
         <string>Hostuj</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="btnPolacz">
        <property name="text">
         <string>Połącz</string>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="horizontalSpacerTop2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <item>
       <widget class="QLabel" name="labelRzuty">
        <property name="text">
         <string>Pozostałe rzuty: 3</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;color:#444;</string>
        </property>
       </widget>
      </item>

     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout" name="diceLayout">
      <property name="spacing">
       <number>12</number>
      </property>

      <item>
       <widget class="QToolButton" name="kostka0">
        <property name="minimumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="maximumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="iconSize">
         <size><width>64</width><height>64</height></size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="kostka1">
        <property name="minimumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="maximumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="iconSize">
         <size><width>64</width><height>64</height></size>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="kostka2">
        <property name="minimumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="maximumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="iconSize">
         <size><width>64</width><height>64</height></size>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="kostka3">
        <property name="minimumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="maximumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="iconSize">
         <size><width>64</width><height>64</height></size>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="kostka4">
        <property name="minimumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="maximumSize">
         <size><width>86</width><height>86</height></size>
        </property>
        <property name="iconSize">
         <size><width>64</width><height>64</height></size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="horizontalSpacerDice">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>

     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="btnRzut">
        <property name="minimumSize">
         <size><width>0</width><height>34</height></size>
        </property>
        <property name="text">
         <string>Rzut</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerBtn">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelInfo">
        <property name="text">
         <string>Kliknij w tabeli w swoją komórkę kategorii, aby zapisać wynik.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#666;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QTableWidget" name="tabela">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
